roles/deploy/handlers/main.yml

```diff
@@ -42,3 +44,10 @@
     state: started
   become: yes
   become_user: "{{ unicorn_user }}"
+
+- name: restart nginx
```

Comment:
Well ansible just uses state: started so could it be either? Would you prefer if these tasks were called start xxx?